### PR TITLE
🎉 add project scaffold

### PR DESCRIPTION
**Pull request by joao-victor-campos**
# Description

Adding scaffold of the project.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Add mock test function.

# Checklist:

- [x] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: requirements.dev.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+pytest
+pytest-cov
+
+# code quality
+black
+isort 
+flake8
+flake8-isort
+flake8-docstrings
+mypy
```
Comment (NicholasBaraldi) [inline requirements.dev.txt:15]: Good requirements!

**Review by NicholasBaraldi — APPROVED** [review]
LGTM

**Review by victorrenop — APPROVED** [review]
lgtm!

**Review by rafaelleinio — APPROVED** [review]
nice work guys!

**Pull request merged**